src/nxdoc/nuxeo-server/installation/tomcat-server-zip/installing-the-nuxeo-platform-on-linux.md

```diff
@@ -1,364 +1,36 @@
-        message: Migration of unmigrated content due to installation of a new plugin
-        version: '12'
-    -
-        author: Solen Guitter
-        date: '2012-09-05 00:55'
-        message: Migrated to Confluence 4.0
-        version: '11'
-    -
-        author: Solen Guitter
-        date: '2012-09-05 00:55'
-        message: ''
-        version: '10'
-    -
-        author: Mathieu Guillaume
-        date: '2012-07-31 21:42'
-        message: Updated ubuntu distribution name to latest LTS
-        version: '9'
-    -
-        author: Thierry Martins
-        date: '2012-04-26 10:04'
-        message: ''
-        version: '8'
-    -
-        author: Solen Guitter
-        date: '2012-03-28 17:32'
-        message: ''
-        version: '7'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 11:47'
-        message: ''
-        version: '6'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 00:29'
-        message: ''
-        version: '5'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 00:03'
-        message: ''
-        version: '4'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 00:01'
-        message: Added toc
-        version: '3'
-    -
-        author: Solen Guitter
-        date: '2011-12-07 11:31'
-        message: ''
-        version: '2'
-    -
-        author: Solen Guitter
-        date: '2011-12-07 11:21'
-        message: ''
-        version: '1'
-
+  date: '2021-01-29'
+  status: ok
+toc: false
 ---
 
-## Checking Requirements
-
-First, have a look at the detailed steps to [check your Java version]({{page page='tomcat-server-zip'}}#java-check) and install it.
-
-## Installing the Nuxeo Platform From the ZIP Archive
-
-### Install the Nuxeo Platform
-
-Download the ZIP archive from the [Nuxeo distribution download site](https://connect.nuxeo.com/nuxeo/site/connect/distributions) and unzip it using your favorite tool.
-
-{{#> callout type='warning' heading='PRIVATE DISTRIBUTION'}}
-You should contact your Nuxeo Administrator or Nuxeo sales representative to get access to this distribution.
-{{/callout}}
-
-### Installation Path
-
-The usual directory into which install the server is `/opt`.
+Check out the tutorials available to tune your Nuxeo Platform installation on Linux:
 
-### What's Next?
+<div class="row" data-equalizer data-equalize-on="medium">
 
-Do you want to evaluate the platform? You can now [start the server]({{page page='server-start-and-stop'}}).
-You want to do a complete installation, that is compatible for a production environment? You should [prepare your environment]({{page page='setup-best-practices'}}) now.
+<div class="column medium-6">
 
-## Recommendations
+{{#> panel type='secondary' match_height='true'}}
+### The essential
 
-### Prevent Swapping
+- [Server Start and Stop]({{page version='' space='nxdoc' page='server-start-and-stop'}})
+- [Setup Best Practices]({{page version='' space='nxdoc' page='setup-best-practices'}})
```

Comment:
I think these links should be in the "Install Nuxeo with the ZIP distribution" page as they're not specific to the Linux distribution but related to the ZIP distribution, independently from any OS.

Reply:
:+1: